README.md

```diff
@@ -63,13 +65,17 @@ In the Arguments section, add five (or more) Arguments Passed On Launch to the b
   ```
   --base-path="/Users/YourName/Desktop/YourProjectFolder"
   ```
-- The path within your project to your asset catalog, without a preceding slash (add multiple times if you have multiple asset catalogs:  
+- The path within your project to your asset catalog, without a preceding slash (add multiple times if you have multiple asset catalogs):  
   ```
   --asset-catalog="Resources/Images.xcassets"
   ```  
   ```
   --asset-catalog="Resources/Media.xcassets"
   ```
+- The prefix for the category method names. If not specified, the default is `ac`. Must be at least 2 characters, and conform to valid method naming rules:
```

Comment:
Hmm, that's a tricky one. However, if someone is using glob patterns in a run script build phase, I'd think they're not overly worried about assigning specific prefixes, since the processing order is essentially undefined. If they wanted to ensure a specific prefix matches a certain asset catalog, they should probably be specific about both in the command line, imho.